### DISCUSSION TITLE
[Feature]  Version-less packages

### DISCUSSION THREAD

**Discussion by couleurm** [ScoopInstaller/Scoop#5430] [Ideas]
#### Is your feature request related to a problem? Please describe.

Yes, I face issues with self updatable apps (e.g Firefox, Discord, VSCode) that are often chromium-based and tend to propose an update before the manifest's bucket ci proposes to update

#### Describe the solution you'd like

Make the manifest based on the version of the installer, not the app (if that is a thing)

#### Describe alternatives you've considered

- For these sort of apps you could also have a manifest (which it's irl is constantly updated, eg discord's) and has no fixed version
- at post_install deserialize the program's config files and disallow updates, then let scoop handle updating it

**Comment by rashil2000** [comment]
> Make the manifest based on the version of the installer, not the app (if that is a thing)

That would be misleading - people install apps based on their version, not the installer's. 

Not to mention that obtaining version information of the installer isn't something I see being possible.
